src/utils/utils_cleaner.h

```diff
@@ -22,11 +22,14 @@ typedef int (*ocf_cleaner_get_item)(struct ocf_cache *cache,
 		void *getter_context, uint32_t item,
 		ocf_cache_line_t *line);
 
+typedef void (*ocf_cleaner_put_item)(struct ocf_cache *cache,
+		void *getter_context, uint32_t item);
+
```

Comment:
It doesn't look like it's meant to be part of this commit.

Reply:
right